release_tester/arangodb/starter/manager.py

```diff
@@ -333,6 +339,7 @@ def send_request(self, instance_type, verb_method,
         http_client.HTTPConnection.debuglevel = 0
         return results
 
+    @step("Crash managed instances and the starter")
```

Comment:
this should be red.

Reply:
Steps must only be red if they have a failed assertion in them, or an uncaught exception. This is a negative test step, so I think it should be green if it worked as expected.